src/components/SelectAssets.tsx

```diff
@@ -25,7 +25,7 @@ export const SelectAssets = ({ selected, handleSelect }: Props) => {
     <button
       type={"button"}
       onClick={handleAssetsSelect}
-      className="max-w-[148px] md:max-w-[210px] bg-white shadow-select-token rounded-full flex justify-between items-center p-1 gap-2.5 dark:bg-black-800 dark:shadow-select-token-dark"
```

Comment:
We should take into account tokens with excessively long symbol names. Long text can stretch this element, disrupting the form layout and breaking the design.

Example:
<img width="853" alt="Screenshot 2024-12-09 at 10 50 08" src="https://github.com/user-attachments/assets/c8a12a78-c1fc-4a36-b0e4-91e4677196f7">

Reply:
Returned the fixed size for token selector and the issue is gone